app/helpers/form_helper.rb

```diff
@@ -203,6 +203,20 @@ def autocomplete_f(f, attr, options = {})
     end
   end
 
+  def byte_size_f(f, attr, options = {})
+    base_id = ActionView::Helpers::Tags::TextField.new(f.object_name, attr, nil, options).send :add_default_name_and_id, {}
+    options[:class] = options[:class].to_s + ' byte_spinner'
+    options[:help_inline] ||= popover('', _("When specifying custom value, add 'MB' or 'GB' at the end. Field is not case sensitive and MB is default if unspecified."))
+    options[:help_block] ||= content_tag(:span, :class => 'maximum-limit hidden') do
+      content_tag(:i, '', :class => 'pficon-warning-triangle-o') +
+      content_tag(:span, ' ' + _('Specified value is higher than recommended maximum'), :class => 'error-message')
+    end
+
+    javascript_include_tag("jquery.ui.custom_spinners.js") +
```

Comment:
This will add the js multiple times on some pages and in the middle of the page, please only include once in pages that need it.